### PR TITLE
[MU4] Fixed view mode availability

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml [hunk 1]
```diff
@@ -62,6 +62,8 @@ Item {
                 return
             }
 
+            menu.model = 0
+            menu.model = model.items
```
Comment (Eism) [inline src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml:66]: binding model.items to the model in the StyledMenu is no longer required

File: src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml [hunk 2]
```diff
@@ -62,6 +62,8 @@ Item {
                 return
             }
 
+            menu.model = 0
```
Comment (Eism) [inline src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml:65]: better to use the clear function for model

Comment (igorkorsukov) [reply src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml:65]: Did not understand the suggestion. Are you suggesting adding a function?
```
function clear() {
    menu.model = 0
}
```

Comment (Eism) [reply src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml:65]: No, I mean menu.model.clear()

Comment (Eism) [reply src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml:65]: No, I mean menu.model.clear()

**Pull request merged**